### DISCUSSION TITLE
Suggested routines based on your communities workouts

### DISCUSSION THREAD

**Discussion by skippybosco** [SteveG/underthebar#6] [Ideas]
Maybe a bit out of scope for what you're currently doing, but I found myself browsing through the workouts from people I follow on my feed in trying to suss out a new routine. 

Got me thinking that there may be a better way to automate this by aggregating all of the workouts and routines from people I follow and building out a routine based on common patterns. 

Any thoughts on how this may work?

**Comment by SteveG** [comment]
well you could aggregate the data, albeit with some pretty heavy api usage, but I'm not real sure what results you could achieve by going through the data.

maybe just applying a filter to the feed could be something without slamming the api too much. ie, filter it by particular exercise or body part to see what other people are doing?

**Reply by skippybosco** [reply]
Yeah it was an attempt at discovering new exercises to bundle a routine from a large pool of workouts.. but I tried it manually over the last week or so and really the existing set programs are as good or better than when I was able to come up.. 

Thanks.
